cinder/tests/unit/volume/drivers/vmware/test_vmware_vmdk.py

```diff
@@ -2571,7 +2571,7 @@ def test_create_temp_backing_from_attached_vmdk(
         vops._create_device_change_for_disk_removal.return_value =\
             [dev_change_disk_remove]
 
-        tmp_name = mock.sentinel.tmp_name
+        tmp_name = "TEMP_BACKING-%s" % mock.sentinel.tmp_name
```

Comment:
This needs to stay `mock.sentinel.tmp_name`. Instead, we have to change line 2597 to use the `"TEMP_BACKING-%s" % tmp_name`

Reply:
yah I'm working on it.  I'm trying to get unit tests to actually run locally, so I can test before I push up